hbase/src/main/scala/akka/stream/alpakka/hbase/HBaseSettings.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.hbase
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.hbase.{ HBaseConfiguration, TableName }
+
+case class HBaseSettings(conf: Configuration = HBaseConfiguration.create(), parallelism: Int, dispatcher: String = "") {
```

Comment:
`final` for all types that are not intended to be subtyped.

Add an object for each settings type with a `create` method mirroring the generated apply-methods (using Java types for collections, options etc) for the Java API